RELEASE_HOWTO.md

```diff
@@ -13,9 +13,14 @@
     Feel free to massage the generated text!
 7.  Commit the changes from steps 4 to 6 in the same commit.
 8.  Tag the commit made in step 7 with an annotated tag with the new version
-    number. "Annotated" means it needs the `-a` flag, and a message. You can
-    give the message in the same go with the `-m`-flag. Sum up the most
-    important thing in the release in the message.
+    number.
+
+    ```
+    git tag -a -m "Mainly bugfixes" v1.5.1
```

Comment:
Thank you for that! 

Just curious in the Argus documentation we have 
```
git tag -m 'Post release bugfixes' v1.5.1
```
as the example. Is that lacking the `-a` flag?

Reply:
`-a` is implied when `-m` is used without other options to add signatures to the tag, so the examples are effectively the same.